config/index.js

```diff
@@ -291,6 +291,12 @@ var conf = convict({
     format: Boolean,
     env: 'LOCKOUT_ENABLED',
     default: false
+  },
+  openIdProviders: {
+    doc: 'root urls of allowed OpenID providers',
+    format: Array,
+    default: [],
+    env: 'OPENID_PROVIDERS'
```

Comment:
Will there be the possibility of additional openid-related config in the future?  If so then maybe it's worth creating a namespace here like `openid.providers` for forwards-compat.